pontoon/teams/views.py

```diff
@@ -77,6 +78,28 @@ def ajax_info(request, locale):
     })
 
 
+@require_POST
+@permission_required_or_403('base.can_manage_locale', (Locale, 'code', 'locale'))
+@transaction.atomic
+def ajax_update_info(request, locale):
+    team_description = request.POST.get('team_info', None)
+
+    # Allow <p> and <br> tags
+    tags = bleach.ALLOWED_TAGS
```

Comment:
Why did you remove the copy altogether? I thought it was a pretty good idea to not edit bleach's variables, to avoid conflicting with other parts of the code that might use it in the future.